<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QPushButton" name="loginState">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>254</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>17</width>
     <height>17</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>17</width>
     <height>17</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="usrHead">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="back">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>180</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>430</width>
     <height>180</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="userName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>190</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>190</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>190</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>190</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="remember">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>258</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoLogin">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>258</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>自动登陆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addUsr">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>300</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="help">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>300</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="registerAccount">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>193</y>
     <width>63</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findPwd">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>222</y>
     <width>63</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>284</y>
     <width>194</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>194</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>194</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="close">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>5</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="min">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>5</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>back</zorder>
  <zorder>usrHead</zorder>
  <zorder>loginState</zorder>
  <zorder>userName</zorder>
  <zorder>pwd</zorder>
  <zorder>remember</zorder>
  <zorder>autoLogin</zorder>
  <zorder>addUsr</zorder>
  <zorder>help</zorder>
  <zorder>registerAccount</zorder>
  <zorder>findPwd</zorder>
  <zorder>login</zorder>
  <zorder>close</zorder>
  <zorder>min</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
